src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -123,15 +125,17 @@ private AnomalyDetectorSettings() {}
             Setting.Property.Dynamic
         );
 
-    public static final Setting<TimeValue> COOLDOWN_MINUTES = Setting
+    public static final Setting<TimeValue> AD_COOLDOWN_MINUTES = Setting
```

Comment:
Is this so we have ability to have a different request timeout between AD and forecasting? Will you add these to the forecaster settings? I don't see a `FORECAST_MAX_RETRY_FOR_UNRESPONSIVE_NODE` or `FORECAST_COOLDOWN_MINUTES`

Reply:
I changed to add a common colldown and max retry setting in TimeSeriesSettings. Please see my changes in src/main/java/org/opensearch/timeseries/TimeSeriesAnalyticsPlugin.java